atd-etl/reassociate_missing_locations/scripts/atd_vzd_update_cr3_locations.py

```diff
@@ -0,0 +1,109 @@
+import json
+import os
+import requests
+import time
+
+
+def make_update() -> dict:
+    """
+    Runs a SQL query against Hasura and tries to assign a
+    location to any Non-CR3s that do not have location yet.
```

Comment:
Old copypasta here too. I think that this one should say CR3s and not Non-CR3s. I'm happy to make a PR for this too since this doesn't affect how this works at all. Not trying to add a bunch of work here!

Reply:
Naw, this is a great time to address these little things - thank you for pointing it out!